code/modules/clothing/masks/transmog.dm

```diff
@@ -17,7 +17,8 @@
 		/obj/item/stack/sheet/animalhide/monkey	=	/obj/item/clothing/mask/morphing/monkey,
 		/obj/item/stack/sheet/animalhide/lizard	=	/obj/item/clothing/mask/morphing/lizard,
 		/obj/item/stack/sheet/animalhide/xeno	=	/obj/item/clothing/mask/morphing/xeno,
-		/obj/item/stack/sheet/animalhide/human	=	/obj/item/clothing/mask/morphing/human)
+		/obj/item/stack/sheet/animalhide/human	=	/obj/item/clothing/mask/morphing/human,
+		/obj/item/weapon/ectoplasm				=	/obj/item/clothing/mask/morphing/ghost)
```

Comment:
It's because of Github's tabbing, the `=` is aligned with the ones above it in DM.